.gitignore

```diff
@@ -1,2 +1,3 @@
 coverage.txt
 irc-slack
+.*.swp
```

Comment:
nitpick: this should probably be in your global gitignore file

Reply:
good point, I'll remove it